_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -302,6 +302,23 @@ uint32_t ConvertRateControlMFX2VAAPI(mfxU8 rateControl, bool bSWBRC)
     }
 }
 
+static mfxU8 ConvertSliceStructureVAAPIToMFX(mfxU8 structure)
```

Comment:
why do we need 2 identical functions in 2 different places? another one is in h264...

Reply:
Where would you put it?

Reply:
Candidates:
* https://github.com/Intel-Media-SDK/MediaSDK/blob/master/_studio/shared/include/mfx_common.h
* https://github.com/Intel-Media-SDK/MediaSDK/blob/master/_studio/shared/include/mfx_utils.h
* https://github.com/Intel-Media-SDK/MediaSDK/blob/master/_studio/mfx_lib/shared/include/mfx_common_int.h

However, if there are plenty of mfx2vaapi and vaapi2mfx function which we reimplement throughout the code, I would create a new file in _studio/shared/include|src: vaapi_utils.h/cpp